README.md

```diff
@@ -19,9 +25,12 @@ https://paybutton.org
 - App will be available at http://localhost:3000.
 - Local changes on source code should trigger a reload immediately.
 
+
 ### Optional configuration
+- To run on production mode, set `ENVIRONMENT=production` on `.env.local`. This would optimize the build so that the app is faster, but local changes will not trigger a reload immediately.
```

Comment:
Change to:

For production, set `ENVIRONMENT=production` in `.env.local`. This optimizes the build for performance and skips the setup of various dev tools (like LiveReload).